python/pyspark/cloudpickle/cloudpickle.py

```diff
@@ -57,7 +57,6 @@
 from .compat import pickle
 from typing import Generic, Union, Tuple, Callable
 from pickle import _getattribute
-from importlib._bootstrap import _find_spec
```

Comment:
@Fokko, can we exclude this file in the linter checking? We're already skipping other lint checks (see `dev/tox.ini`) and this is usually exactly matched to the release of cloudpickle.

Reply:
Yes, good point! I've moved the flake8 configuration from the command line arguments to the `tox.ini`.